examples/todolist/tests/bindings/test_todolist.kts

```diff
@@ -0,0 +1,15 @@
+import uniffi.todolist.*
+
+val todo = TodoList()
+todo.addItem("Write strings support")
+
+assert(todo.getLast() == "Write strings support")
+
+todo.addItem("Write tests for strings support")
+
+assert(todo.getLast() == "Write tests for strings support")
+
+val entry = TodoEntry("Write bindings for strings as record members")
+todo.addEntry(entry)
```

Comment:
Could you please add another todo item in each of these examples with the text `"Test Ünicode hàndling"` or similar, to test our handling of non-ascii characters? Of course feel free to select your own unicode characters or even put in some emoji to really test it out 😆